### DISCUSSION TITLE
Issue in availing Student benefits

### DISCUSSION THREAD

**Discussion by HoneyVerma22** [community/community#55435] [GitHub Education]
### Hi everyone,

I'm verified for students developer pack.
But when I'm trying to avail my Microsoft Azure student pack it's showing unable to verify your university ID.
What to do now I tried many times.

**Comment by samus-aran** [comment]
Hi @HoneyVerma22 

If this [Azure verification page](https://signup.azure.com/studentverification?offerType=1) does not work then we would suggest you reach out to [Azure Support](https://support.microsoft.com/en-us/topic/2315e669-8b1f-493b-5fb1-d88a8736ffe4) if you are having problems redeeming your Azure benefits.

Let us know how you get on 👍
